react/features/base/util/helpers.js

```diff
@@ -144,3 +144,30 @@ export function reportError(e: Object, msg: string = '') {
     console.error(msg, e);
     window.onerror && window.onerror(msg, null, null, null, e);
 }
+
+/**
+ * Adds alpha to a color css string.
+ *
+ * @param {string} color - The color string either in rgb... Or #... Format.
+ * @param {number} opacity -The opacity(alpha) to apply to the color. Can take a value between 0 and 1, including.
+ * @returns {string} - The color with applied alpha.
+ */
+export function setColorAlpha(color: string, opacity: number) {
+    if (!color) {
+        return `rgba(0, 0, 0, ${opacity})`;
+    }
+
+    let b, g, r;
+
+    if (color.startsWith('rgb')) {
+        [ r, g, b ] = color.split('(')[1].split(')')[0].split(',').map(c => c.trim());
+    } else if (color.startsWith('#')) {
```

Comment:
What if the color is expressed with the shorthand `#fff` for example?

Reply:
Added handler for shorthand colors, also guarding for any potential errors: send original color value on error parsing.